coredns/metadata.csv

```diff
@@ -0,0 +1,21 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+coredns.response_code_count,gauge,,response code,,number of responses per zone and rcode,1,coredns,response code
+coredns.response_code_count.count,count,,response code,,instant number of responses per zone and rcode,1,coredns,response code
+coredns.proxy_request_count,gauge,,request,,query count per upstream.,1,coredns,proxy request count
+coredns.proxy_request_count.count,count,,request,,query count per upstream at a moment,1,coredns,proxy request count
+coredns.cache_hits_count,gauge,,hits,,Counter of cache hits by cache type,1,coredns,cache hit
+coredns.cache_hits_count.count,count,,hits,,Counter of cache hits by cache type at a moment,1,coredns,cache hit
```

Comment:
unit_name should be `hit` not `hits`

https://docs.datadoghq.com/developers/metrics/#units